Lib/test/test_dict.py

```diff
@@ -1312,6 +1312,19 @@ def test_reversed(self):
         self.assertEqual(list(r), list('dcba'))
         self.assertRaises(StopIteration, next, r)
 
+    def test_reverse_iterator_for_empty_dict(self):
```

Comment:
Please add a reference to the bpo in a comment and a short explanation of the crash.